.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,34 @@
+name: Bug Report
+description: Submit a report if you believe you've found a bug in Dymos.
+labels: ["bug"]
+assignees:
+  - robfalck
```

Comment:
Do you really want everything auto-assigned to you?

Reply:
I do this on the OpenMDAO side just to help make sure I see whatever comes in and it's worked well there.  I usually just use that as a "now its my job to add this issue to the project and deal with it as necessary"